irods/test/collection_test.py

```diff
@@ -15,13 +15,29 @@
 from irods.test.helpers import my_function_name, unique_name
 from irods.collection import iRODSCollection
 
+RODSUSER = 'nonadmin'
 
 class TestCollection(unittest.TestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        adm = helpers.make_session()
+        if adm.users.get(adm.username).type != 'rodsadmin':
+            raise cls.WrongUserType('Must be an iRODS admin to run tests in class {0.__name__}'.format(cls))
+        cls.logins = helpers.iRODSUserLogins(adm)
+        cls.logins.create_user(RODSUSER, 'abc123')
+
+
+    @classmethod
+    def tearDownClass(cls):
+        # TODO: Skipping this will result in an interpreter seg fault for Py3.6 but not 3.11; why?
```

Comment:
weird.  do we need an issue # to add to this TODO?

Reply:
Yes. Will take care of that.

Reply:
Done.